src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -535,6 +535,60 @@ An GoogleComputeEngine NodeSource can now be added using the *Create Node Source
 
 WARNING: When ProActive server is stopped, GoogleComputeEngine instances will be automatically deleted. And when ProActive server is restarted, the infrastructure will be recovered as per the previous settings. The required GoogleComputeEngine instances will be re-created and re-configured.
 
+==== Azure Infrastructure
+
+The Resource Manager allows to deploy ProActive nodes on the Microsoft Azure cloud. This infrastructure will create and manage virtual machines (VMs) with your custom Azure image and host ProActive nodes on them. Currently, the Azure infrastructure supports the creation of Linux VMs; support for Windows VMs is under development. In this section we will guide you through the requirements and configuration needed to setup an Azure infrastructure.
```

Comment:
Not sure but in Friday demo, you said that you can use a custom as well as an existing/predefined Azure image.

Reply:
Under developpement --> is that necessary ?

Reply:
Thanks Amine, fair comment and question:

- Friday demo was about the Azure **Scale Set** infra, for which I'm currently writing the corresponding doc. The Azure Scale Set supports standard and custom images, as well as windows and linux. The Scale Set Infra should in the mid-term become our _de facto_ Azure infrastructure. This is another _basic_ Azure infrastructure, which can only handle custom images (standard images are no longer supported).

- Windows images are no longer supported in this infra (they were before). We have standardized the way to create the boot script so that all other infras use it as well (gce, aws, openstack,...), and such script exist only for linux at the moment. However, there is still a field for _imageOSType_, and references to the fact that windows images are an option.
Given that support for windows is expected soon, I did not want to delete such references. What do you suggest, @aminelouati?